### ISSUE TITLE
Rocha doesn't work with compilers

### ISSUE THREAD

**Issue by bhough** [bahmutov/rocha#7]
Attempting to do the following:

```rocha ./test/**/**/*.jsx --recursive --compilers js:babel-core/register```

This returns a syntax error since it doesn't recognize the ES6 syntax. Have also tried this with a coffeescript compiler with the same result. Would love to use this but need compiler support.

**Comment by bahmutov** [comment]
Agreed, Mocha does this using wrapper functions at the startup, see _mocha and mocha files in https://github.com/mochajs/mocha/tree/master/bin

Not sure when I will get to this,
